.ahub/sam/exclude.txt

```diff
@@ -5,6 +5,17 @@
 # Eigen
 /ONE/compiler/nnc/backends/soft_backend/code_snippets/eigen.def
 
+# Frontend tools for testing and debugging
```

Comment:
AFAIK, if these tools are not needed for release, they should be removed from preset, not here.

Reply:
> they should be removed from preset, not here.

`preset` items are used also for coverage test which needs testing.

Reply:
What is the range of coverage test?

In case of static analysis, the range is "source codes which are needed to create release package".
If the range of coverage test is "all source codes in the repository", the script for coverage test should be changed IMHO.

Reply:
> AFAIK, `preset` items are used also for coverage test which needs testing.

@hseok-oh hold me `coverage` list is in another file --> I need to revise this file... -_-;
From my long time ago memory, test modiules in `preset` list are added as some modules need them while build configure without checking target existance. we may need to revise them later if we are to remove them from `preset` list.

Reply:
> What is the source code range of coverage test?

`infra/scripts/compiler_modules.sh` provides modules for release package + test modules for test these modules

Reply:
I think some of these can be removed